litmus-portal/frontend/README.md

```diff
@@ -75,115 +50,68 @@ frontend
-```
 npm run unit:ci
-```
 
-<br>
-
-#### 🛠️ **Build the project**
-
-```
+🛠️ Build the project
 npm run build
-```
-
-Builds the app for production to the `build` folder.<br>
-It correctly bundles React in production mode and optimizes the build for the best performance.
-The build is minified and the filenames include the hashes.
+Builds the app for production to the build folder.
+It correctly bundles React in production mode and optimizes the build for the best performance. The build is minified and the filenames include the hashes.
 
-<br>
 
-#### 🏴🏴‍☠️ **Lint your code**
-
-```
+🏴🏴‍☠️ Lint your code
 npm run lint
-```
-
-ESLint statically analyzes your code to quickly find problems.<br>
-You would be able to see any lint errors in the console based on the rules set on `.eslintrc`
+ESLint statically analyzes your code to quickly find problems.
+You would be able to see any lint errors in the console based on the rules set on .eslintrc
 
-<br>
 
-#### ♻️ **Format your Code**
-
-```
+♻️ Format your Code
 npm run format
-```
-
-Formats your document according to the prettier guidelines and then does a `eslint --fix`
-Builds the app for production to the `build` folder.<br>
-It correctly bundles React in production mode and optimizes the build for the best performance.
-`Prettier` for formatting and `linters` for catching bugs! 🐛
-<br><br>
+Formats your document according to the prettier guidelines and then does a eslint --fix Builds the app for production to the build folder.
+It correctly bundles React in production mode and optimizes the build for the best performance. Prettier for formatting and linters for catching bugs! 🐛
 
-### `3. Tech Stack`
 
-<br>
+3. Tech Stack
 
-<img src = 'https://image.flaticon.com/icons/svg/919/919853.svg' width='30'/> <img src = 'https://external-content.duckduckgo.com/iu/?u=http%3A%2F%2Fwww.codeproject.com%2FKB%2FHTML%2F843860%2FTypeScript.png&f=1&nofb=1' width='30'/> <img src = 'https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fcdn.volaresystems.com%2FImages%2FPosts%2F2019%2F12%2Feslint_logo_text.png&f=1&nofb=1' width='30'/> <img src = 'https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ffaridoon.com%2Fwp-content%2Fuploads%2F2019%2F08%2FMaterialUILogo.png&f=1&nofb=1' width='30'/> <img src = 'https://image.flaticon.com/icons/svg/919/919851.svg' width='30'/> <img src = 'https://image.flaticon.com/icons/svg/919/919856.svg' width='30'/> <img src = 'https://upload-icon.s3.us-east-2.amazonaws.com/uploads/icons/png/3556671901536211770-512.png' width='30'/> <img src = 'https://upload.wikimedia.org/wikipedia/commons/thumb/1/17/GraphQL_Logo.svg/1200px-GraphQL_Logo.svg.png' width='30'/> <img src = 'https://d2eip9sf3oo6c2.cloudfront.net/tags/images/000/001/216/thumb/apollo-seeklogo.com_%281%29.png' width='30'/><img src = 'https://cdn.iconscout.com/icon/free/png-512/redux-283024.png' width='30'>
-<br><br>
+        
 
-### `4. Frontend Contribution Guidelines`
 
-<br>
+4. Frontend Contribution Guidelines
```

Comment:
This is also supposed to be a bold list item which is linked with the Table of Contents.